### PR TITLE
Adding default domain and policies

### PR DESCRIPTION
**Pull request by larkinscarvalho**

### CODE REVIEW

**Comment by larkinscarvalho** [comment]
Merge wont work. Because, merge tries to update with new values. If it finds 'Default', it will update the row with new id. Updating primary key causes the violation. Working on a different approach.

File: src/main/resources/META-INF/create.sql [hunk 1]
```diff
@@ -32,8 +32,12 @@ alter table SECURITY_GROUP_INTERFACE_POLICY add constraint if not exists FK_SGI_
                     
 alter table SECURITY_GROUP_INTERFACE_POLICY add constraint if not exists FK_SGI_POLICY_POLICY foreign key (policy_fk) references POLICY;
 
-MERGE INTO Domain VALUES (1, 'Default');
+alter table domain MODIFY column id BIGINT auto_increment;
```
Comment (arvindn05) [inline src/main/resources/META-INF/create.sql]: can we do this a part of create table

Comment (larkinscarvalho) [reply src/main/resources/META-INF/create.sql]: Yes, we can. But, it will break the existing use of plugin.

**Comment by larkinscarvalho** [comment]
Testing done!

**Pull request merged**